frontend/src/component/project/Project/ProjectOverviewChangeRequests.tsx

```diff
@@ -42,16 +44,32 @@ const ChangeRequestCount = styled(Typography)(({ theme }) => ({
 export const ProjectOverviewChangeRequests: FC<{ project: string }> = ({
     project,
 }) => {
+    const { isChangeRequestConfiguredInAnyEnv } =
+        useChangeRequestsEnabled(project);
+    const { data: projectInsights } = useProjectInsights(project);
```

Comment:
here I read all project insights only to grab change requests info. I'm also considering a dedicated resource (can be in enterprise change requests). One advantage of warming up the insights resource is that when we go to insights we already have it. The drawback is that we overload project overview with the full insights query